.ci.yaml

```diff
@@ -64,6 +64,18 @@ targets:
       - codesign/**
       - .ci.yaml
 
+  - name: Mac arm_codesign
```

Comment:
in what way will this build be any different from the intel one?

Reply:
oh wait, nvm, i see cpu under properties, lol

Reply:
umm I guess a `cpu:arm64` will have this cipd be built on an arm64 machine?